docs/content/RadioButton.md

```diff
@@ -0,0 +1,92 @@
+---
+title: RadioButton
+description: Use radio buttons when users need to select one option from a list
+status: Alpha
+source: https://github.com/primer/react/blob/main/src/RadioButton.tsx
+storybook: '/react/storybook?path=/story/forms-radio-button--default'
+---
+
+import {ComponentChecklist} from '../src/component-checklist'
```

Comment:
ComponentChecklist is globally available in markdown files now 🎉 
```suggestion
```

Reply:
Thanks. Removed.